fixtures/datasources/main.go

```diff
@@ -136,36 +145,35 @@ func getEnvOrDefault(key, defaultValue string) string {
 
 func main() {
 	logger.Infof("Setting up")
-	if err := prepareS3E2E(s3Fixtures); err != nil {
+	if err := prepareS3E2E(context.Background(), s3Fixtures); err != nil {
 		logger.Errorf("error setting up %v", err)
 	}
 }
 
 var (
 	s3Fixtures = S3Fixture{
-
 		CreateBuckets: []string{
 			"tests-e2e-1",
 			"tests-e2e-2",
 		},
 		Files: []S3FixtureFile{
 			{
 				Bucket:      "tests-e2e-1",
-				Filename:    "/pg/backups/date1/backup.zip",
+				Filename:    "pg/backups/date1/backup.zip",
```

Comment:
Had to remove the leading slashes from the object names because now it throws an error

> level=error msg=error setting up failed to put object /pg/backups/date1/backup.zip to bucket tests-e2e-1: operation error S3: PutObject, https response error StatusCode: 400, RequestID: 17608288C83D0B51, HostID: 5b550a06-4d30-43cb-a932-73cf24838d5a, api error XMinioInvalidObjectName: Object name contains a leading slash.

Reply:
Can we remove them in the check function so that either is ok